server/server.js

```diff
@@ -21,7 +23,7 @@ function Server() {
   this.subs = {};
 }
 
-MiniEventEmitter.mixin(Server);
```

Comment:
It works fine either before or after, but I will standardize on putting it after the definition of the main class in the module, which in this case is Server.